Sources/Build/PkgConfig.swift

```diff
@@ -0,0 +1,161 @@
+        for path in (searchPaths + envSearchPaths) {
+            let pcFile = Path.join(path, "\(name).pc")
+            if pcFile.isFile {
+                return pcFile
+            }
+        }
+        throw PkgConfigError.CouldNotFindConfigFile
+    }
+    
+    mutating func load() throws {
+        cFlags = [String]()
+        libs = [String]()
+        try parser.parse()
+        if let cFlags = parser.cFlags {
+            // FIXME: handle spaces in paths.
+            self.cFlags += cFlags.characters.split(separator: " ").map(String.init)
+        }
+        if let libs = parser.libs {
+            // FIXME: handle spaces in paths.
+            self.libs += libs.characters.split(separator: " ").map(String.init)
+        }
+        
+        if(parser.dependencies.isEmpty) {
+            return
+        }
+        
+        for dep in parser.dependencies {
+            var pkg = try PkgConfig(name: dep)
+            try pkg.load()
+            self.cFlags += pkg.cFlags
+            self.libs += pkg.libs
+        }
+    }
+}
+
+private struct PkgConfigParser {
+    let pcFile: String
+    var variables = [String: String]()
+    var dependencies = [String]()
+    var cFlags: String?
+    var libs: String?
+    
+    enum Token {
+        case CFlats(String)
+        case Libs(String)
+        case Dependencies(String)
+        case Variable(name: String, value: String)
+    }
+    
+    init(pcFile: String) {
+        self.pcFile = pcFile
+    }
+    
+    mutating func parse() throws {
+        let file = File(path: self.pcFile)
+        for line in try file.enumerate() {
+            if !line.characters.contains(":") && line.characters.contains("=")  {
+                let equalsIndex = line.characters.index(of: "=")!
+                let name = line[line.startIndex..<equalsIndex]
+                let value = line[equalsIndex.successor()..<line.endIndex]
+                variables[name] = resolveVariables(value)
+            } else if line.hasPrefix("Requires: ") {
+                dependencies = parseDependencies(value(line: line))
+            } else if line.hasPrefix("Libs: ") {
+                libs = resolveVariables(value(line: line)).chomp()
+            } else if line.hasPrefix("Cflags: ") {
+                cFlags = resolveVariables( value(line: line)).chomp()
+            }
+        }
+    }
+    
+    func parseDependencies(_ depString: String) -> [String] {
+        let exploded = depString.characters.split(separator: " ").map(String.init)
+        let operators = ["=", "<", ">", "<=", ">="]
+        var deps = [String]()
+        var skipNext = false
+        for depString in exploded {
+            if skipNext {
+                skipNext = false
+                continue
+            }
+            if operators.contains(depString) {
+                skipNext = true
+            } else {
+                deps.append(depString)
+            }
+        }
+        return deps
+    }
```

Comment:
This method isn't very robust, it accepts a lot of things which (I think) are invalid ("a = = b"), and it would be good to have some comments here about what it is trying to do.

Also, IMHO here is a nicer way to write this kind of loop which makes it more readable:
```
    var it = exploded.makeIterator()
    while let arg = it.next() {
        // Check if we have found an operator.
        if operators.contains(arg) {
          // We should have a version number next.
          guard let arg = it.next() else {
             // ... error ...
             continue
          }
        } else {
          // ...
        }
```